cmd/agent/install_mac_os.sh

```diff
@@ -5,7 +5,7 @@
 
 # Datadog Agent install script for macOS.
 set -e
-install_script_version=1.5.0
+install_script_version=1.5.1
```

Comment:
Nit: since we still haven't released 1.5.0 as such I think we could leave the version unchanged.

```suggestion
install_script_version=1.5.0
```